core/pkg/models/patches.go

```diff
@@ -0,0 +1,19 @@
+package models
+
+type PatchOp string
+
+const (
+	OpAdd     PatchOp = "add"
```

Comment:
This is a global const so I propose to use a more context related name like PatchAdd, PatchRemove,...